pkg/parser/pipelineyml/visitor_snippet.go

```diff
@@ -1,14 +1,25 @@
 package pipelineyml
 
 import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/flosch/pongo2.v3"
+
 	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/pkg/expression"
 )
 
 const (
 	Snippet            = "snippet"
 	SnippetLogo        = "http://terminus-paas.oss-cn-hangzhou.aliyuncs.com/paas-doc/2020/10/22/410935c6-e399-463a-b87b-0b774240d12e.png"
 	SnippetDisplayName = "嵌套流水线"
 	SnippetDesc        = "嵌套流水线可以声明嵌套的其他 pipeline.yml"
+
```

Comment:
`SnippetLogo` address needs to be obtained using configuration

Reply:
no need now

Reply:
this file will be delete totally soon. after all modules migrate to erda, then adjust it.